doc/source/development.rst

```diff
@@ -216,16 +216,28 @@ Format and Linting
 
 **Running linter locally:** To run the Python linter on a specific file, run
 something like ``flake8 ray/python/ray/worker.py``. You may need to first run
-``pip install flake8``.
+``pip install flake8==3.7.7``.
```

Comment:
want to also include the relevant version of `yapf` and `flake8-quotes`?

Reply:
Also, the best way to do all of this is probably to just use the pre-push hooks.